lib/uplink/common.go

```diff
@@ -0,0 +1,13 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package uplink
+
+import "storj.io/storj/pkg/storj"
+
+// Encryption specifies an individual bucket's encryption choices
+type Encryption struct {
```

Comment:
how is this different from BucketOpts? or why does it not have `EncryptionScheme` on it as a field? could they be the same struct?

Reply:
It's been renamed to Encryption 👍